### DISCUSSION TITLE
C# physics server

### DISCUSSION THREAD

**Discussion by gameboycjp** [godotengine/godot-proposals#7184] [Scripting]
I'm trying to create a physics server extension, and the original library I'm implementing is in C#. I can access the needed classes from C#, but there's no way to change the physics engine at runtime. My first assumption of what I'd need for this is C# support in GDExtension, but I'm not sure. Do GDExtensions exist below modules? Are they able to access functionality of modules?

**Comment by AThousandShips** [comment]
For support questions and help please turn to the other [community channels](https://godotengine.org/community), this is for feature discussion, not a help desk.

**Comment by GeorgeS2019** [comment]
@gameboycjp 
https://github.com/godotengine/godot-proposals/discussions/7024

**Comment by GeorgeS2019** [comment]
@gameboycjp 
If you rephrase your title to C# GDExtension binding to Jolt physics, Godot future Physics, this is a legitimate FEATURE worth discussion
https://github.com/godotengine/godot-proposals/discussions/5161
